posts/2025-04-08-25.0.0.4-beta.adoc

```diff
@@ -0,0 +1,250 @@
+
+[source,yaml]
+----
+  probes:
+    startup:
+      exec:
+        command:
+          - /bin/sh
+          - '-c'
+          - file=/output/health/started; cat $file
+      failureThreshold: 5
+      initialDelaySeconds: 15
+      periodSeconds: 15
+      timeoutSeconds: 5
+    liveness:
+      exec:
+        command:
+          - /bin/sh
+          - '-c'
+          - file=/output/health/live; x=$(cat $file) && mts=$(stat --format=%.3Y $file | tr -d ".") && if [ $(expr $(date +%s%3N) - $mts) -gt 15000 ] ; then (exit 1) ; else (exit 0) ; fi
+      failureThreshold: 3
+      initialDelaySeconds: 10
+      periodSeconds: 15
+      timeoutSeconds: 5
+    readiness:
+      exec:
+        command:
+          - /bin/sh
+          - '-c'
+          - file=/output/health/ready; x=$(cat $file) && mts=$(stat --format=%.3Y $file | tr -d ".") && if [ $(expr $(date +%s%3N) - $mts) -gt 15000 ] ; then (exit 1) ; else (exit 0) ; fi
+      failureThreshold: 3
+      initialDelaySeconds: 10
+      periodSeconds: 15
+      timeoutSeconds: 5
+----
+
+Notice that in the liveness and readiness probes, the `periodSeconds` is set to 15, or 1.5x the `fileUpdateInterval`, and the check that the file was last modified within the 15 seconds, or the `-gt 15000` part of the command.
+
+
+[#support]
+== InstantOn Support for J2EEManagement, AppClientSupport and WsSecurity
+
+Open Liberty link:{url-prefix}/docs/latest/instanton.html[InstantOn] provides fast startup times for MicroProfile and Jakarta EE applications. With InstantOn, your applications can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features. InstantOn uses the Checkpoint/Restore In Userspace (link:https://criu.org/[CRIU]) feature of the Linux kernel to take a checkpoint of the JVM that can be restored later. InstantOn supports link:{url-prefix}/docs/latest/instanton.html#supported-features[a subset of Open Liberty features]. As of the 25.0.0.4-beta release, the following features are enhanced to support InstantOn.
+
+-  link:{url-prefix}/docs/latest/reference/feature/j2eeManagement-1.1.html[J2EE Management 1.1 (j2eeManagement-1.1)]
+- link:{url-prefix}/docs/latest/reference/feature/appClientSupport-1.0.html[Application Client Support for Server 1.0 (appClientSupport-1.0)]
+- link:{url-prefix}/docs/latest/reference/feature/appClientSupport-2.0.html[Jakarta Application Client Support for Server 2.0 (appClientSupport-2.0)]
+- link:{url-prefix}/docs/latest/reference/feature/wsSecurity-1.1.html[Web Service Security 1.1 (wsSecurity-1.1)]
+
+[#run]
+=== Try it now
+
+To try out these features, update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 21, Java SE 17, Java SE 11, and Java SE 8.
```

Comment:
```suggestion
To try out these features, update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 24, Java SE 21, Java SE 17, Java SE 11, and Java SE 8.
```